Source/Core/Common/TraversalClient.h

```diff
@@ -0,0 +1,87 @@
+// This file is public domain, in case it's useful to anyone. -comex
+
+#pragma once
+
+#include "Common.h"
+#include "Thread.h"
+#include "TraversalProto.h"
+#include <functional>
+#include <memory>
+#include <list>
+#include "enet\enet.h"
+
```

Comment:
Again, these should be relative to the `Core` directory. Also standard library includes come before everything else. These also should be alphabetical.